src/test/com/aca/homework/week10/user/repository/UserRepositoryTest.java

```diff
@@ -0,0 +1,111 @@
+package com.aca.homework.week10.user.repository;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.function.Executable;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class UserRepositoryTest {
+
+    private final CrudRepository testSubject = new UserRepository();
```

Comment:
because you have one testSubject instance, your tests are co-dependant. one test execution may affect on another one. please use setUp method